src/main/java/cinema/controller/MovieSessionController.java

```diff
@@ -47,13 +48,16 @@ public void create(@RequestBody MovieSessionRequestDto movieSessionRequestDto) {
                 .collect(Collectors.toList());
     }
 
-    @PutMapping
-    public void update(@RequestBody MovieSessionRequestDto movieSessionRequestDto) {
-        movieSessionService.update(movieSessionMapper.toMovieSessionEntity(movieSessionRequestDto));
+    @PutMapping("/{id}")
+    public void update(@PathVariable("id") Long id,
+                       @RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        MovieSession movieSession = movieSessionMapper.toMovieSessionEntity(movieSessionRequestDto);
+        movieSession.setId(id);
+        movieSessionService.update(movieSession);
     }
 
-    @DeleteMapping
-    public void delete(@RequestBody MovieSessionRequestDto movieSessionRequestDto) {
-        movieSessionService.delete(movieSessionMapper.toMovieSessionEntity(movieSessionRequestDto));
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable("id") Long id) {
```

Comment:
```suggestion
    public void delete(@PathVariable Long id) {
```
If names match - no need to specify Path Variable name

Reply:
Fixed